src/profile.c

```diff
@@ -2040,6 +2071,14 @@ profile_transcode_work(profile_chain_t *prch,
   props.tp_vbitrate   = profile_transcode_vbitrate(pro);
   props.tp_abitrate   = profile_transcode_abitrate(pro);
   strncpy(props.tp_language, pro->pro_language ?: "", 3);
+  
+  if(!pro->pro_src_vcodec) {
+	  strcpy(props.tp_src_vcodec, "");
+  } else if(!strncasecmp("Any",pro->pro_src_vcodec,3)) {
```

Comment:
This looks wrong. Use the key match, not the language string , so the first condition should be:

  if(!pro->pro_src_vcodec || pro->pro_src_vcodec[0] == '\0') {